src/utils/locale.ts

```diff
@@ -155,15 +207,20 @@ export function getSupportedLocale(locale: string): string {
     return "no";
```

Comment:
we may need to move this block above the  `.includes `check to make sure if the user parses `nb` we return `no` or we replace `nb` with `no` as t9nLocales array.